dbms/src/Functions/tests/gtest_func_return_type.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <AggregateFunctions/AggregateFunctionFactory.h>
+#include <AggregateFunctions/AggregateFunctionSum.h>
+#include <DataTypes/DataTypeDecimal.h>
+#include <DataTypes/DataTypeNullable.h>
+#include <TestUtils/AggregationTestUtils.h>
+#include <TestUtils/mockExecutor.h>
+#include <TestUtils/TiFlashTestBasic.h>
+
+#include <tuple>
+#include <vector>
+
+namespace DB
+{
+namespace tests
+{
+
+class ExecutorFuncReturnTypeTestRunner : public DB::tests::AggregationTest
```

Comment:
move to `dbms/src/AggregateFunctions/tests`

Reply:
> move to `dbms/src/AggregateFunctions/tests`

okk